app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,52 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+  version :thumb do
+  process :resize_to_fill => [50, 50]
```

Comment:
PR #49 with Rubocop support hasn't landed yet, but I'd recommend the 1.9-style hash syntax here in preparation:

```ruby
process resize_to_fill: [50, 50]
```